pingcap/diag
# add slow query retriever

### What problem does this PR solve? 
support checking slow log

### What is changed and how it works?
- add slow query log retriever.
- add some accumulators to calculate statistics.

### Check List 

Tests 

 - Unit test

Code changes

 - Add slow query log retriever.
 - Add `includes` flag to `check` command.
 - File fetcher use slow query retriever to load slow_query.log.
 - Add toml tag to tidb config data structure.
